src/hotspot/share/prims/universalUpcallHandler.cpp

```diff
@@ -46,6 +46,9 @@ extern struct JavaVM_ main_vm;
 struct UpcallContext {
   Thread* attachedThread;
 
+  #ifdef _AIX
+    UpcallContext() {}
+  #endif
```

Comment:
I'd prefer for this to not be AIX specific. Adding an explicit empty constructor seems harmless enough - with a comment of course e.g.

UpCallContext() {} // Explicit constructor to avoid potential xlc compiler bug

@JornVernee what do you think?

Reply:
Yeah, that seems fine to me as well.

Reply:
+1